tensorflow/compiler/mlir/tosa/transforms/legalize_tf.cc

```diff
@@ -1653,6 +1661,52 @@ LogicalResult ConvertTFPadOp::matchAndRewrite(Operation* op,
   return success();
 }
 
+LogicalResult ConvertTFResizeBilinearOp::matchAndRewrite(
+    Operation* op, PatternRewriter& rewriter) const {
+  auto tf_resize_op = cast<TF::ResizeBilinearOp>(op);
+
+  RankedTensorType output_type =
+      tf_resize_op.getResult().getType().dyn_cast<RankedTensorType>();
+  // Not a ranked tensor output
+  if (!output_type) return failure();
+
+  bool align_corners = tf_resize_op.align_cornersAttr().getValue();
```

Comment:
nit: inline into call site?

Reply:
Thanks will do. I'd assume same comment for "half_pixel_centers" 1 line below?